scripts/docgen.lua

```diff
@@ -71,6 +81,22 @@ Default config:
 
 ]]
 
+local lsp_section_template_md = [[
+## {{config_name}}
+
+{{preamble}}
+
+**Snippet to enable the language server:**
+```lua
+require'lspconfig'.{{config_name}}.setup{}
+```
+{{commands}}
+**Default config:**
```

Comment:
would prefer to minimize the differences between txt vs md. the only real difference is the title. don't need bold here or above.